docs/user-guide/installation/f23-.rst

```diff
@@ -127,12 +116,15 @@ Server
 #. Install the Pulp server, task workers, and their dependencies. For Pulp installations that use
    Qpid, install Pulp server using::
 
-    $ sudo yum groupinstall pulp-server-qpid
+    $ sudo yum install pulp-server pulp-selinux python-gofer-qpid python-qpid qpid-tools
```

Comment:
Not that long ago, `pulp-server` now carries a `Reuqires:` statement for `pulp-selinux` as its own dep.